1.1/profile.ttl

```diff
@@ -13,8 +13,8 @@
     a prof:Profile ;
     owl:versionIRI <http://www.opengis.net/def/geosparql/1.1> ;
     dcterms:created "2020-10-11"^^xsd:date ;
-    dcterms:modified "2020-12-22"^^xsd:date ;
-    dcterms:issued "2021"^^xsd:gYear ;
+    dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
+    dcterms:modified "2021-06-13"^^xsd:date ;
     dcterms:publisher <https://ror.org/00fsdxs93> ;
```

Comment:
let's try to use the same modeling pattern throughout all artefacts

Reply:
Fixed in Commit https://github.com/opengeospatial/ogc-geosparql/pull/152/commits/747ee942174a35a8f3e2d676b0b4671d4c86fd7f